### PR TITLE
Add info about deploying puppeteer to Heroku

### PR DESCRIPTION
**Pull request by jontewks**
Fixes https://github.com/GoogleChrome/puppeteer/issues/758

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -135,3 +135,11 @@ Run the container by passing `node -e "<yourscript.js content as a string>` as t
 
 There's a full example at https://github.com/ebidel/try-puppeteer that shows
 how to run this setup from a webserver running on App Engine Flex (Node).
+
+## Running Puppeteer on Heroku
+
+Running puppeteer on Heroku requires some additional dependencies that aren't included on the Linux box that Heroku spins up for you. To add the dependencies on deploy, all you need to do is add the puppeteer heroku buildpack to the list of buildpacks for your app under Settings > Buildpacks.
```
Comment (ebidel) [inline docs/troubleshooting.md]: s/puppeteer/Puppeteer x 2

s/heroku/Heroku

Comment (ebidel) [reply docs/troubleshooting.md]: Remove "all you need to do is"

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -135,3 +135,11 @@ Run the container by passing `node -e "<yourscript.js content as a string>` as t
...
+
+The url for the buildpack is https://github.com/jontewks/puppeteer-heroku-buildpack
+
+When you click add buildpack, simply paste that url into the input, and click save. On the next deploy, your app will also install the dependencies that puppeteer needs to run.
```
Comment (ebidel) [inline docs/troubleshooting.md]: Puppeteer

**Review by ebidel — CHANGES_REQUESTED** [review]
Thanks! Just some nits.

**Comment by jontewks** [comment]
Thanks @ebidel, updated and squashed together.

**Review by ebidel — APPROVED** [review]

**Pull request merged**